src/routes/devRoute.js

```diff
@@ -29,6 +30,7 @@ if (env !== 'production') {
   router.post('/update-user-score-phase1', auth.dev, updateUserScorePhase1);
   router.post('/update-user-score-phase2', auth.dev, updateUserScorePhase2);
   router.post('/remove-activity', auth.dev, removeActivity);
+  router.post('/news-process', auth.dev, newsProcessDevProcess);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new route `/news-process` is added correctly. However, please ensure that the `newsProcessDevProcess` function has proper error handling and validation for the incoming request data to avoid potential issues.
```diff
- router.post('/news-process', auth.dev, newsProcessDevProcess);
+ router.post('/news-process', auth.dev, (req, res, next) => {
+   try {
+     // Validate request data here
+     newsProcessDevProcess(req, res, next);
+   } catch (error) {
+     next(error);
+   }
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->